{"name": "test_successful_logout", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Timeout 10000ms exceeded.\n=========================== logs ===========================\nwaiting for navigation to \"**/*office*\" until 'load'\n============================================================", "trace": "page = <Page url='https://liner.dstepanyuk.dev.smte.am/auth/'>\nconfig = {'apiUrl': 'https://liner.dstepanyuk.dev.smte.am/api/', 'baseUrl': 'https://liner.dstepanyuk.dev.smte.am/auth/', 'brow...password': 'wrong_password'}, 'valid_user': {'email': 'dstepanyuk@southmedia.io', 'password': 'yND-jC2-rWz-qqJ'}}, ...}\n\n    @pytest.fixture\n    def authenticated_page(page, config):\n        from pages.login_page import LoginPage\n        login_page = LoginPage(page, config[\"baseUrl\"])\n        login_page.navigate()\n        creds = config[\"credentials\"][\"valid_user\"]\n        login_page.login(creds[\"email\"], creds[\"password\"])\n>       page.wait_for_url(\"**/*office*\", timeout=10000)\n\ntests\\conftest.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nvenv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:9161: in wait_for_url\n    self._sync(\nvenv\\Lib\\site-packages\\playwright\\_impl\\_page.py:575: in wait_for_url\n    return await self._main_frame.wait_for_url(**locals_to_params(locals()))\nvenv\\Lib\\site-packages\\playwright\\_impl\\_frame.py:233: in wait_for_url\n    async with self.expect_navigation(\nvenv\\Lib\\site-packages\\playwright\\_impl\\_event_context_manager.py:33: in __aexit__\n    await self._future\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    async def continuation() -> Optional[Response]:\n>       event = await waiter.result()\nE       playwright._impl._errors.TimeoutError: Timeout 10000ms exceeded.\nE       =========================== logs ===========================\nE       waiting for navigation to \"**/*office*\" until 'load'\nE       ============================================================\n\nvenv\\Lib\\site-packages\\playwright\\_impl\\_frame.py:207: TimeoutError"}, "start": 1741790640741, "stop": 1741790640741, "uuid": "13d100a1-b37e-4281-ad0a-85f3704e88d9", "historyId": "05fcf6eb05425c1014f1f00d4e7f2e41", "testCaseId": "05fcf6eb05425c1014f1f00d4e7f2e41", "fullName": "tests.test_auth.TestAuthentication#test_successful_logout", "labels": [{"name": "story", "value": "Successful logout"}, {"name": "feature", "value": "Authentication"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "@pytest.mark.usefixtures('page')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestAuthentication"}, {"name": "host", "value": "DESKTOP-US86AEC"}, {"name": "thread", "value": "16000-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}